### PR TITLE
Add tests for braille translators.

### PR DESCRIPTION
**Pull request by jakubl7545**
This PR adds unit tests for braille translators - ueb and nemeth. For both coverage is above 90%.

### CODE REVIEW

File: latex_access/nemeth.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def frac(self,input,start,rting=()):
             else:
                 transnum=self.translate(numerator[0],(rting[0]+numerator[2],rting[1]+1))
                 transden=self.translate(denominator[0],(rting[0]+denominator[2],rting[1]+len(transnum)+2))
-                translation="?"+transum+"/"+transden+"#"
+                translation="?"+transnum+"/"+transden+"#"
```
Comment (lukaszgo1) [inline latex_access/nemeth.py:118]: Please revert this change - PR's introducing tests should not touch code they test.

File: latex_access/tests/test_nemeth.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        self.assertEqual(self.braille.sqrt('{2x}', 0), ('>2x}', 4))
+        self.assertEqual(self.braille.sqrt('{16}', 0, (1, 0)), ('>16', 4))
+
+    def test_frac(self):
+        """Tests translations of fractions."""
+        self.assertEqual(self.braille.frac('{1}{2}', 0), ('1/2', 6))
+        self.assertEqual(self.braille.frac('{1}', 0), ('?1/#', 3))
+        self.assertEqual(self.braille.frac('{x}{y}', 0), ('?x/y#', 6))
+        #Disable because of typo in tested code.
+        #self.assertEqual(self.braille.frac('{2x}{3y}', 0, (1, 1)), ('?2x/3y#', 8))
```
Comment (lukaszgo1) [inline latex_access/tests/test_nemeth.py:47]: The correct way to disable the tests where we know the cause of the failure is to move the code to a separate test, and mark it with `@unittest.expectedFailure`. The same goes for the misspell in `UEB`. In the current state we may easily forgot to uncomment the test whenever the code is fixed.

File: latex_access/tests/test_ueb.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        self.assertTrue(self.braille.letterSign('{ B }', 2))
+        self.assertTrue(self.braille.letterSign('{ }', 0))
+        self.assertTrue(self.braille.letterSign('{,}', 2))
+        self.assertTrue(self.braille.letterSign('{b,}', 1))
+        self.assertTrue(self.braille.letterSign('{,b }', 2))
+        self.assertTrue(self.braille.letterSign(' b', 1))
+
+    def test_lowerLetter(self):
+        """Tests translations of lower letters."""
+        self.braille.lastnumber = None
```
Comment (lukaszgo1) [inline latex_access/tests/test_ueb.py:146]: In the code `lastnumber` is always set to a digit, and never to `None`, Either change this to a number, or add an explanatory comment, which clarifies why `None` is appropriate here.

File: latex_access/tests/test_ueb.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+        self.assertEqual(self.braille.lowerLetter('b', 0), (';b', 0))
+        self.assertEqual(self.braille.lowerLetter('{b}', 2), ('b', 2))
+        self.assertEqual(self.braille.lowerLetter('{ABc}', 4), (',c', 4))
+
+    def test_upperLetter(self):
+        """Tests translations of upper letters."""
+        self.braille.lastnumber = None
```
Comment (lukaszgo1) [inline latex_access/tests/test_ueb.py:153]: Same here W.R.T `None` vs an int.

**Review by lukaszgo1 — CHANGES_REQUESTED** [review]
In general looks good. When the remarks  below are addressed this should be ready to be merged. By the way I've added some additional tests, in places where it seemed easy to increase coverage.

**Comment by jakubl7545** [comment]
@lukaszgo1 All done. Also managed to increase coverage for ueb slightly.

**Review by lukaszgo1 — APPROVED** [review]
LGTM. I'm going to merge this.

**Pull request merged**